### PR TITLE
fix: when quitting chrome exit the process

### PR DESCRIPTION
**Pull request by elevatebart**
In component-testing when quitting the chrome instance, the process needs to exit.

## How to test?

- Start component testing
- Type Ctrl-Q to quit chrome

The process should exit normally

### CODE REVIEW

File: packages/server-ct/index.ts [hunk 1]
```diff
@@ -27,7 +27,11 @@ export const start = async (projectRoot: string, args: Record<string, any>) => {
 
     return openProject.create(projectRoot, args, options)
     .then((project) => {
-      return openProject.launch(browser, spec)
+      return openProject.launch(browser, spec, {
+        onBrowserClose: () => {
+          process.exit()
```
Comment (JessicaSachs) [inline packages/server-ct/index.ts:32]: Looks good to me. I might've added a CT specific debug log, but it's not a deal breaker.  The current DEBUG logs look like this:

```
  cypress:launcher chrome exited: { code: 0, signal: null } +5s
  cypress:server:browsers killing browser process +4s
  cypress:server:browsers:chrome closing remote interface client +4s
✨  Done in 10.87s.
```

**Review by JessicaSachs — APPROVED** [review]

File: packages/server-ct/index.ts (hunk 1 above)
Comment (JessicaSachs) [reply packages/server-ct/index.ts:36]: we already have a debugger...  like `import { debug as debugFn } from debug` and then `const debug = debugFn('cypress:server-ct:browsers')` yknow?

Comment (JessicaSachs) [reply packages/server-ct/index.ts:36]: I'm fine to keep it as a callback.

Comment (elevatebart) [reply packages/server-ct/index.ts:36]: The event emit would still be emitted in the callback.

**Review by lmiller1990 — APPROVED** [review]
That was easy

**Pull request merged**